op-e2e/README.md

```diff
@@ -19,3 +19,15 @@ runtime, but others cannot be changed or else it will result in
 broken tests. Any changes to `devnetL1.json` should result in
 rebuilding the `.devnet` artifacts before the new values will
 be present in the `op-e2e` tests.
+
+## Running tests
+Consult the [Makefile](./Makefile) in this directory. Run, e.g.:
+
+```bash
+make test-http
+```
+
+### Troubleshooting
+If you encounter errors:
+* ensure you have the latest version of foundry installed (run `pnpm update:foundry` or `foundryup`)
+* try running `pnpm clean`
```

Comment:
We could also suggest to first try only deleting the `forge-artifacts` directory.

Reply:
[40f6838](https://github.com/ethereum-optimism/optimism/pull/9380/commits/40f6838842e3fa64f7a1ed5f8e8aa6e06c1902c2)